Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/webDriverSingleton/Driver.java

```diff
@@ -5,21 +5,13 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
-public class WebDriverSingleton {
+public class Driver {
 	private static WebDriver driver;
 
-	private WebDriverSingleton() {
-	}
-
-	public static WebDriver getWebDriverInstance() {
+	public static WebDriver initBrowser() {
```

Comment:
usually in singleton we use method name getInstance() or something like that. It's common practice